spec/discoverGranules/DiscoverGranules.input.template.json

```diff
@@ -0,0 +1,55 @@
+{
+  "cumulus_meta": {
+    "message_source": "sfn",
+    "state_machine": "",
+    "execution_name": ""
+  },
+  "meta": {
+    "workflow_name": "DiscoverGranules",
+    "stack": "",
+    "buckets": {
```

Comment:
Should this be ```"buckets": {
      "internal": "cumulus-test-sandbox-internal",
      "private": "cumulus-test-sandbox-private"
    },```?

Reply:
I followed the trend of the other specs. They start blank in the ..input.template.json and then they get populated after running the test into the ...input.json. 
They get populated from the workflows.yml

Reply:
Sorry! I had been looking at the generated file. My bad!